ompi/mca/pml/yalla/pml_yalla_request.h

```diff
@@ -25,15 +25,16 @@ struct pml_yalla_base_request {
     ompi_request_t               ompi;
     mca_pml_yalla_convertor_t    *convertor;
     int                          flags;
-    /* overlaps with base of send/recv
-     * In ISO C90, you would have to give contents a length of 1,
-     * which means either you waste space or complicate the argument to malloc.
-     * Note:
-     *  - 1 was the portable way to go, though it was rather strange
-     *  - 0 was better at indicating intent, but not legal as far as
-     *  the Standard was concerned and supported as an extension by some compilers (including gcc)
-     */
-    mxm_req_base_t               mxm_base[1];
+
+    /* FIXME: how other compilers define C89/C90? */
+#if defined(__STRICT_ANSI__)
```

Comment:
@jsquyres yeah, that's what I though - was writing a comment while you responded :)